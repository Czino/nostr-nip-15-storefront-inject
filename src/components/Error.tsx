export const Error = ({ children }: React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>) => (
    <div className="bg-red-700 text-white p-4 rounded text-center">{children}</div>
)
