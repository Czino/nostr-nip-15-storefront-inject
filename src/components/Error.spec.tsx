import { render } from '@testing-library/react'
import { Error } from './Error'

describe('Error', () => {
    it('renders correctly', () => {
        const { asFragment } = render(<Error>Oops</Error>)
        expect(asFragment()).toMatchSnapshot()
    })
})
