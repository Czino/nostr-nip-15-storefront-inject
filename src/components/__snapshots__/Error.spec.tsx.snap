// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-red-700 text-white p-4 rounded text-center"
  >
    Oops
  </div>
</DocumentFragment>
`;
